apps/etterlatte-samordning-vedtak/README.md

```diff
@@ -12,10 +12,16 @@ Det må foreligge et tjenestepensjonsforhold i Tjenestepensjonsregisteret som gj
 
 ## API
 
-| Endepunkt                      | Responstype         | Beskrivelse                                                                                                                           |
-|:-------------------------------|---------------------|:--------------------------------------------------------------------------------------------------------------------------------------|
-| /api/vedtak?virkFom=YYYY-MM-DD | Samordningsvedtak[] | Henter ut vedtaksinformasjon for gitt person fra og med gitt dato. <br/> Fødselsnummeret angis i en `fnr`-header                      |
-| /api/vedtak/{nav-vedtak-id}    | Samordningsvedtak   | Henter ut informasjon om et spesifikt vedtak. VedtaksIDen kommer fra samordningskøen hvor det varsles løpende om vedtak som gjøres.   |
+| Endepunkt                      | Headers        | Responstype         | Beskrivelse                                                                                                                         |
+|:-------------------------------|----------------|---------------------|:------------------------------------------------------------------------------------------------------------------------------------|
+| /api/vedtak?virkFom=YYYY-MM-DD | fnr <br/> tpnr | Samordningsvedtak[] | Henter ut vedtaksinformasjon for gitt person fra og med gitt dato.                                                                  |
+| /api/vedtak/{nav-vedtak-id}    | tpnr           | Samordningsvedtak   | Henter ut informasjon om et spesifikt vedtak. VedtaksIDen kommer fra samordningskøen hvor det varsles løpende om vedtak som gjøres. |
+
+| Header | Beskrivelse                      |
+|--------|----------------------------------|
+| tpnr   | kallende tjenestepensjonsordning |
+| fnr    | fødselsnummer til aktuell person |
```

Comment:
Er andre typar identifikatorar også aktuelle, eller må du ha norsk fnr for å kunne få tenestepensjon?

Reply:
er ikke kjent med noe annet, kanskje @HelgeJanbu har noe input

Reply:
er ikke kjent med noe annet, kanskje @HelgeJanbu har noe input